collections/crowdsecurity/supabase-compose.md

```diff
@@ -0,0 +1,24 @@
+## Supabase Docker Compose collection
```

Comment:
Couldnt we just name the files just supabase? dont think adding the compose adds any value

Reply:
There would be hopefully another collection for supabase AMIs from https://github.com/supabase/postgres .

Reply:
@LaurenceJJones with Shivam's answer, do you think it makes sense to name it like this? r would you have an alternative to propose? 
lets be sure and resolve this dicussion